apps/communications/contacts/views/details/js/boot.js

```diff
@@ -0,0 +1,51 @@
+/* global DetailsUI, DetailsController, LazyLoader, fbLoader, ContactsService */
+'use strict';
+
+window.addEventListener('DOMContentLoaded', function() {
+  LazyLoader.load(['/shared/js/l10n.js']).then(function() {
+    LazyLoader.load([
+      document.getElementById('view-contact-details')
+    ]).then(function() {
+      // TODO Add if needed
+    });
+  });
+});
+
+window.onload = function() {
+  var dependencies = [
+    '/contacts/services/contacts.js',
+    '/contacts/js/activities.js',
+    '/shared/js/l10n_date.js',
+    '/shared/js/contact_photo_helper.js',
+    '/shared/js/contacts/contacts_buttons.js',
+    '/shared/js/text_normalizer.js',
+    '/shared/js/contacts/utilities/dom.js',
+    '/shared/js/contacts/utilities/templates.js',
+    '/shared/js/contacts/import/utilities/misc.js',
+    '/contacts/js/match_service.js',
+    '/contacts/views/details/js/details_ui.js',
+    '/contacts/views/details/js/details_controller.js',
+    '/contacts/js/fb_loader.js'
```

Comment:
Do we need to load this? Why? This is impacting the performance due to internally is loading more files.